tests/test_vectors/discrete_gauss_9999999999999999999999.json

```diff
@@ -1,56 +1,56 @@
-        -6306079673435155346359,
-        -3923441892847593054161,
-        -15343913081587795695543,
-        6358156846193247612690,
-        7419825606177014954794,
-        5188566641087329402683,
-        -6758430220371638235303,
-        -4790100404639726653559,
-        -16014248963401012891620,
-        -3070071895986196677964,
-        -5848874668833745227877,
-        -18435830148517064469147,
-        -17848621274303274471834,
-        5282430088172496749183,
-        8576946480142860155499,
-        -2828646354360531511240,
-        -942850098400867509851,
-        9425651555597696445218,
-        -2539281423953275968251,
-        11187378493262462376834,
-        -6116678358159812997083,
-        -3436435729276833415628,
-        200162077639402997265,
-        1902018077580024126206,
-        3514979444113021823498,
-        -10598978329592410882809,
-        -4432092109314175600675,
-        -9433517318412068100406,
-        3751331871729952870001,
-        -2157632349890764345554,
-        636693770470222913362,
-        -13762970481594739356390,
-        -24485079813076283116966,
-        -214843260419243736734,
-        -8207508139915530781866,
-        11326059754769588764304
+        7826646418794482000000,
+        4044429794334089400000,
+        -13887062284591121000000,
+        4816851335312673000000,
+        1899078604295677400000,
+        -7819872990828025000000,
+        1164017821807881500000,
+        -25360379570365623000000,
+        5906637163630390000000,
+        3730592807287609400000,
+        1737147847266613600000,
+        18593764766679059000000,
+        22724295990478920000000,
+        -4396717652132314000000,
+        -7933138830987044000000,
+        12204708418993710000000,
+        10716232788607488000000,
+        -7423575920998904000000,
+        -274262846168968760000,
+        -24595460253341310000000,
+        1880635641243101800000,
+        10823060437484008000000,
+        -2525077352000184000000,
+        -1421364839539048700000,
+        -2648842672480402300000,
+        -7783156811031001000000,
+        -1831198454606609500000,
+        905920470298728600000,
+        -8805882598094077000000,
+        2949974625887522000000,
+        13071000629486423000000,
+        1311702736683394000000,
+        -14044034250430824000000,
+        1421736709768854000000,
+        14824744520414923000000,
+        10752031849750700000000,
+        -522118577625103100000,
+        2006618532306506000000,
+        -7573105805904098000000,
+        -14482966128638640000000,
+        -11408400475022384000000,
+        -17555433966245180000000,
+        -6120185353438187000000,
+        -4778266138627264000000,
+        -19325342657405320000000,
+        3725950229250476300000,
+        -7977400544074383000000,
+        2738166449787592000000,
+        -1321521809406566300000
     ],
     "seed": "000102030405060708090a0b0c0d0e0f",
     "std_denom": 1,
-    "std_num": 9999999999999999999999
```

Comment:
@divergentdave this results from passing the file through `jq`.

Reply:
Hmm, we should back this line out. It looks like serde is choking on parsing it, as it's expecting to parse a `u64` out of a numeric literal. Setting that aside, we may get cascading differences in the test's noise outputs, due to changing how many times a rejection sampling loop runs. JSON has limited interoperability guarantees for integers outside of the MIN_SAFE_INTEGER to MAX_SAFE_INTEGER range, but if Python and Serde both handle this fine, we can do a little hand holding around jq, and otherwise get away with this schema for test vectors.

Reply:
Done.